src/catalog/src/builtin.rs

```diff
@@ -5369,341 +5375,303 @@ JOIN root_times r USING (id)",
 pub const MZ_SHOW_DATABASES_IND: BuiltinIndex = BuiltinIndex {
     name: "mz_show_databases_ind",
     schema: MZ_INTERNAL_SCHEMA,
-    sql: "CREATE INDEX mz_show_databases_ind
-IN CLUSTER mz_introspection
+    sql: "CLUSTER mz_introspection
```

Comment:
Can you add a comment to this struct field that indicates what exactly it should be?